site/content/docs/development/architecture.md

```diff
@@ -0,0 +1,144 @@
+# Architecture and Concepts
+
+## Overview
+
+Cartographer is an open-sourced Supply Chain Choreographer for Kubernetes. Cartographer provides a set of Kubernetes
+controllers and CRDs that allow a platform operator to create an application platform by specifying Supply Chains and 
+application Delivery workflows.
+
+## Concepts
+
+### Blueprints
+
+Blueprints are a list of templates (called resources) and how they depend upon each other. It forms the dependency graph of
+your supply chain or delivery.
+
+The dependencies are formed by specifying which resource(s) are used as inputs.
+
+Blueprints consist of:
+* A **selector** to match owners
+* **Parameters** to pass to all resources 
+* **Resources**: 
+  * A **templateRef** pointing to the template for the resource
+  * **Parameters** to pass to the template
+  * **Inputs**, which specify dependencies for the template
+
+![Blueprint](../img/blueprint.jpg)
```

Comment:
![Blueprint](./img/blueprint.jpg)

All of the image tags go back one folder too many, and so don't display the image.

Reply:
They work perfectly when rendered.  Check the preview https://deploy-preview-422--elated-stonebraker-105904.netlify.app/docs/development/architecture